grc/core/platform.py

```diff
@@ -133,7 +133,12 @@ def build_library(self, path=None):
         self.cpp_connection_templates.clear()
         self._block_categories.clear()
 
-        with Cache(Constants.CACHE_FILE, version=self.config.version) as cache:
+        try:
```

Comment:
How can this fail?

Reply:
my mental model was that GRC is something we perspectively want to move out of the GR tree and make less tightly version-coupled.

In the wake of that, depending on the very recent addition of `gr.paths` (or gr.paths.persistent) is still sensible (because we're still having GRC in-tree), but falling back gracefully in this place makes it easier to use "new" GRC with "old" GRC in the future.